.eslintrc.json

```diff
@@ -1,32 +1,3 @@
 {
-	"extends": ["eslint:recommended", "plugin:prettier/recommended"],
-	"plugins": ["prettier"],
-	"rules": {
-		"prettier/prettier": "error"
-	},
-	"env": { "es6": true },
-	"parserOptions": {
-		"ecmaVersion": 2018,
-		"sourceType": "module"
-	},
-	"overrides": [
-		{
-			"files": ["src/**/*.ts"],
-			"parser": "@typescript-eslint/parser",
-			"parserOptions": { "project": "./tsconfig.json" },
-			"plugins": ["@typescript-eslint"],
-			"extends": [
-				"eslint:recommended",
-				"plugin:@typescript-eslint/eslint-recommended",
-				"plugin:@typescript-eslint/recommended",
-				"prettier/@typescript-eslint"
-			]
-		},
-		{
-			"files": ["test/**/*.ts"],
-			"env": {
-				"jest": true
-			}
-		}
-	]
+	"extends": ["./node_modules/@sofie-automation/code-standard-preset/eslint/main", "prettier"]
```

Comment:
This immediately seems a bit unelegant referencing the file system directly, but I guess we can't use the module name and have it automatically resolved here then?